devel/gcovr/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+name                gcovr
+version             8.2
+revision            0
+categories          devel python
+license             BSD
+maintainers         nomaintainer
+
+description         A utility for generating summarizing code coverage results from gcov
+
+long_description    Gcovr provides a utility for managing the use of the GNU \
+                    gcov utility and generating summarized code coverage results.
+
+homepage            https://gcovr.com
+
+checksums           rmd160  f54851529dd23c2d442f34c6000e3d9d6bce5642 \
+                    sha256  9a1dddd4585d13ec77555db5d6b6a31ee81587ea6fc604ff9fcd232cb0782df5 \
+                    size    176540
+
+python.default_version 313
+python.pep517       yes
+
+depends_build-append \
+                    port:py${python.version}-hatchling \
```

Comment:
set the "python.pep517_backend" to the correct value and drop the explicit dependency here